client/src/pages/Home/FooterIcon/FooterIcon.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './FooterIcon.css';
+import { GrFacebookOption, GrTwitter, GrInstagram  } from "react-icons/gr";
+
+function FooterIcon() {
+  return (
+    <div className='footericon_container'>
+        <div  className='footericon_iconstext' >
+           <div className='footericon_icons'>
+           <a href="https://www.facebook.com">  <GrFacebookOption className='footericon_icon' /></a>
+           <a href="https://www.twitter.com">    <GrTwitter className='footericon_icon' /></a>
```

Comment:
there is an underline appears under the icons . a corresponding CSS should be added to remove it